Examples/Physics_applications/capacitive_discharge/analysis_dsmc.py

```diff
@@ -57,4 +55,4 @@
 
 density_data = np.load( 'ion_density_case_1.npy' )
 print(repr(density_data))
-assert np.allclose(density_data, ref_density, rtol=0.01)
+assert np.allclose(density_data, ref_density, rtol=1e-3)
```

Comment:
How small can `rtol` be? The density should be exactly reproducible.

Reply:
I'll leave it to the default (1e-5) and we can see if it that causes any issues.